src/main/java/polyglot/config/CommandLineArgs.java

```diff
@@ -50,6 +50,29 @@
   @Option(name = "--tls_ca_cert_path", metaVar = "<path>")
   private String tlsCaCertPath;
 
+  // *************************************************************************
+  // * Initial step towards the migration to "polyglot <command> [flagz...]" *
+  // *************************************************************************
+  @Option(name = "--command", metaVar = "<e.g. list_services>")	// Expand to support many commands
+  private String commandArg;
+  
+  // TODO: Move to a "list_services"-specific flag container
+  @Option(name = "--service_filter", metaVar = "service_name", 
+		  usage="Filters services to those containing this string")
+  private String serviceFilterArg;
+  
+  // TODO: Move to a "list_services"-specific flag container
+  @Option(name = "--method_filter", metaVar = "method_name", 
+		  usage="Filters methods on a service to those containing this string")
```

Comment:
I think that was an issue with tabs. What layout is standard? I have currently changed it to this:
```
  // TODO: Move to a "list_services"-specific flag container
  @Option(name = "--method_filter", metaVar = "method_name", 
          usage="Filters methods on a service to those containing this string")
  private String methodFilterArg;
```